inception-recommendation/src/test/java/de/tudarmstadt/ukp/inception/recommendation/service/RecommendationServiceImplIntegrationTest.java

```diff
@@ -162,20 +193,40 @@ private Project createProject(String aName)
         return testEntityManager.persist(project);
     }
 
-    public AnnotationLayer createAnnotationLayer()
+    private AnnotationLayer createAnnotationLayer()
     {
         AnnotationLayer layer = new AnnotationLayer();
         layer.setEnabled(true);
-        layer.setName("annotation type name");
+        layer.setName(NamedEntity.class.getName());
         layer.setReadonly(false);
-        layer.setType("test type");
+        layer.setType(NamedEntity.class.getName());
         layer.setUiName("test ui name");
         layer.setAnchoringMode(false, false);
        
         return testEntityManager.persist(layer);
     }
 
-    public AnnotationFeature createAnnotationFeature(AnnotationLayer aLayer, String aName)
+    private User createUser()
+    {
+        User user = new User();
```

Comment:
`return new User()`?

Reply:
Why  a method for this at all?

Reply:
I did it to conform to the other setup functions, if you need to add manipulate the user then its already there for you.